recipes/userspace-rcu/all/test_package/test_package.cpp

```diff
@@ -0,0 +1,141 @@
+#include <benchmark/benchmark.h>
```

Comment:
We usually try to keep the test_package simplified, is it possible to have a simple main that called a few key functions just to test the linkage?

Reply:
Sure! I copied this from our internal code as I'm not the original consumer for the package and "just grabbed something". Can definitely remove this dependency if that's ideal.